singularity-eos/eos/eos_eospac.hpp

```diff
@@ -41,42 +41,46 @@ using namespace eos_base;
 
 // Only really works in serial
 // Not really supported on device
+#define SG_PIF_NOWARN \
+#pragma nv_exec_check_disable \
+PORTABLE_INLINE_FUNCTION
+
```

Comment:
should this be hidden behind a check for portability strategy?

Reply:
how's that @Yurlungur ?

Reply:
Yep this is fine. We may want to move this macro into ports-of-call at some point, but that can wait for another day.